policyengine_canada/variables/gov/provinces/bc/tax/income/credits/bccatc/bc_climate_action_tax_credit.py

```diff
@@ -0,0 +1,32 @@
+from policyengine_canada.model_api import *
+
+
+class bc_climate_action_tax_credit(Variable):
+    value_type = float
+    entity = Household
+    label = "British Columbia Climate Action Tax Credit after reduction"
```

Comment:
How the gov page does it
```suggestion
    label = "British Columbia climate action tax credit after reduction"
```